server/routers/loginRouter.js

```diff
@@ -6,9 +6,10 @@ const loginController = require('../controllers/loginController');
 // check for a cookie before any request
 // create a middleware that handles checking for cookies
 
-router.post('/',
-loginController.createUser,
-(req, res) => res.status(200).json({}));
+router.post('/', loginController.createUser, (req, res) => { 
+    console.log('saved user to database');
+    return res.status(200).json(res.locals.newUser);
```

Comment:
Be careful not to return newUser because this would contain this individuals username and password, information the frontend doesn't need. Perhaps change status to 201 for created content.